src/pages/development/components/app-server.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Application Server | Commerce PHP Extensions
+description: Learn about the Application Server architecture and how to ensure compatibility with custom extensions.
+keywords:
+  - Extensions
+edition: ee
+---
+
+<InlineAlert variant="info" slots="text" />
+
+Available in [2.4.7-beta](https://experienceleague.adobe.com/docs/commerce-operations/release/notes/adobe-commerce/2-4-7.html) only.
+
+# Application Server for GraphQL APIs
+
+The [Application Server for GraphQL APIs](https://experienceleague.adobe.com/docs/commerce-operations/performance-best-practices/performance-best-practices/application-server.html) enables Adobe Commerce to maintain state among GraphQL API requests. The Application Server, which is built on the Open Swoole extension, operates as a process with worker threads that handle request processing. By preserving a bootstrapped application state among GraphQL API requests, the Application Server enhances request handling and overall product performance. API requests become significantly more efficient.
```

Comment:
Hi @jeff-matthews, I want us to provide details on how GraphQL performance is actually improved with application server and add the following statement here: "As a result, GraphQL request response time can be reduced up to 30%."

Reply:
Thanks for reviewing @chernenm. I pushed a change to capture that statement.